src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -96,23 +136,36 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime советуют использовать при записи даты в базу данных.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        ZoneOffset zoneOffSet = ZoneOffset.of("+02:00");
+        return OffsetDateTime.of(localTime, zoneOffSet);
     }
 
     /**
      * Дана строка в виде "yyyymmdd".
      * Необходимо вернуть Optional даты в LocalDate формате
      */
     public Optional<LocalDate> parseDate(String date) {
-        return Optional.empty();
+        LocalDate givenDate;
+        try {
+            givenDate = LocalDate.parse(date, formatterParseDate);
```

Comment:
there is already a predefined formatter in DateTimeFormatter class

Reply:
done